python/lib/db/query/site.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import select
+from sqlalchemy.orm import Session as Database
+
+from lib.db.model.candidate import DbCandidate
+from lib.db.model.session import DbSession
+from lib.db.model.site import DbSite
+
+
+def try_get_site_with_psc_id_visit_label(db: Database, psc_id: str, visit_label: str):
+    """
+    Get a session from the database using its candidate CandID and visit label, or return `None`
```

Comment:
the doc says it uses the candID but the query uses the PSCID.

Reply:
Whoops, that is indeed an oversight on my part, I'll be fixing that !